domains/justin.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "My personal Page",
+    "repo": "https://github.com/justinjohnymathew/justinjohnymathew.github.io",
+    "owner": {
+      "username": "justinjohnymathew",
+      "email": "justinjohny.jj@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://justinjohnymathew.github.io/",
```

CNAME must be a host. Full urls dont work

```suggestion
      "CNAME": "justinjohnymathew.github.io",
```